lasagne/layers/recurrent.py

```diff
@@ -49,10 +49,18 @@
 >>> batchsize, seqlen, _ = l_inp.input_var.shape
 >>> l_lstm = LSTMLayer(l_inp, num_units=num_units)
 >>> # In order to connect a recurrent layer to a dense layer, we need to
->>> # flatten the first two dimensions (our "sample dimensions"); this will
->>> # cause each time step of each sequence to be processed independently
->>> l_shp = ReshapeLayer(l_lstm, (-1, num_units))
->>> l_dense = DenseLayer(l_shp, num_units=num_classes)
+>>> # process each time step of each sequence to be independently.
+>>> # this will be taken care of by setting the `num_leading_axes` to -1
+>>> l_dense = DenseLayer(l_lstm, num_units=num_classes, num_leading_axes=-1)
+>>> # An example involving Feeding the output of `LSTMLayer` to `Conv1DLayer` 
```

Comment:
I am not sure I understood this comment very well, do you mean I should create a new network from the start using an InputLayer for the second example?